frontend/src/scripts/scripts.js

```diff
@@ -1,5 +1,27 @@
 $(document).foundation();
 
-$(document).ready(function() {
+$(window).on("load", function() {
+    
+    // rotating the cubes on the front hero panel using a timer
+    
+    var x = true;
```

Comment:
Could `var x` be renamed to a more meaningful name? Makes it easier for someone else to come along and read your code once the project has been handed over :smile: